<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>402</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Sources of requests</string>
    </property>
    <widget class="QLineEdit" name="inSourceCount">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>241</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of sources</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inSourceMax">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inSourceMin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>251</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimum time interval</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>251</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximum time interval</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>361</width>
      <height>142</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Queue</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>200</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Queue length</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inBufSize">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Processing devices</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>251</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of devices</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inDeviceCount">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Exponential distribution Lambda parameter</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="inDeviceLambda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>281</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>202</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Simulation mode</string>
    </property>
    <widget class="QRadioButton" name="buttonAutomatic">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>182</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="buttonStep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>202</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Step-by-step</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start simulation</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
